extensions/azcli/src/typings/az-ext.d.ts

```diff
@@ -171,12 +171,26 @@ declare module 'az-ext' {
 			}
 		},
 		status: {
-			readyReplicas: string, // "1/1"
-			state: string, // "Ready",
-			logSearchDashboard: string, // https://127.0.0.1:30777/kibana/app/kibana#/discover?_a=(query:(language:kuery,query:'custom_resource_name:miaa1'))
-			metricsDashboard: string, // https://127.0.0.1:30777/grafana/d/40q72HnGk/sql-managed-instance-metrics?var-hostname=miaa1-0
-			primaryEndpoint?: string // "10.91.86.39:32718"
-			runningVersion: string // "v1.5.0_2022-04-05"
+			endpoints: {
+				logSearchDashboard: string, // https://127.0.0.1:30777/kibana/app/kibana#/discover?_a=(query:(language:kuery,query:'custom_resource_name:miaa1'))
+				metricsDashboard: string, // https://127.0.0.1:30777/grafana/d/40q72HnGk/sql-managed-instance-metrics?var-hostname=miaa1-0
+				mirroring: string,
+				primaryEndpoint?: string // "10.91.86.39:32718"
+			},
+			highAvailability: {
+				lastUpdateTime: string,
+				mirroringCertificate: string
+			},
+			lastUpdateTime: string,
+			observedGeneration: string,
```

Comment:
Are all these new properties going to exist on every request? Make sure to mark any optional ones as optional themselves